Lombiq.NodeJs.Extensions/scripts/skip-styles.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @summary A helper script to determine whether to skip running the styles pipeline.
+ * @description Returns 0 if the styles pipeline should be skipped, else 1.
+ */
+
+const { access } = require('node:fs/promises');
+const path = require('path');
+
+(async function main() {
+    try {
+        const stylesSourceDirectory = path.resolve('..', '..', process.env.SCSS_SOURCE);
+        await access(stylesSourceDirectory);
+        process.exit(1);
+    }
+    catch (_) {
+        process.stdout.write('Skipping styles');
+        process.exit(0);
+    }
```

Comment:
Why are you returning 1 on success and 0 on failure? This is very abnormal.

Reply:
Because I'm checking whether I should **skip** processing **styles**.

Reply:
It's still incredibly alien and unnatural to return failure code in the try and success code in the catch.

What's wrong with `const styleDirectoryExists = fs.existsSync(path.resolve('..', '..', process.env.SCSS_SOURCE))` that returns Boolean instead of throws?

Reply:
Checking the existence of a path and verifying access to it is not the same. Maybe this is me being overly cautious but when I read about `fs.access` it actually made a lot of sense to use that instead of `fs.exists` (or *sync).

The other decision I made was to use `access` over `accessSync` because I thought that's preferable because the executing thread doesn't block on the I/O operation. And that's how I ended up with this "incredibly alien and unnatural" piece of code.

Reply:
Currently if the path is inaccessible it will skip the step, failing silently. I don't think this is a desirable behavior.

Reply:
That's a very good point!